experiments/Compute.Experiments/AzureRM.Compute.Experiments.Tests.ps1

```diff
@@ -16,25 +16,21 @@ $vmComputerUser = $credentials.vmUser;
 $password = ConvertTo-SecureString $vmComputerPassword -AsPlainText -Force;
 $vmCredential = New-Object System.Management.Automation.PSCredential ($vmComputerUser, $password);
 
-New-AzVm -Name MyVM -Credential $vmCredential -WhatIf
-
-# $job = New-AzVm -Name MyVMA -Credential $vmCredential -AsJob
-# Receive-Job $job
-
-# exit
-
-# $vm = New-AzVm
-# $vm = New-AzVm -Credential $vmCredential
-$vm = New-AzVm -Name MyVMA1 -Credential $vmCredential -ResourceGroupName Crocodile
-# $vm = New-AzVm -Name MyVMA
-
-$vm
-
-# Write-Host "<async>"
-# $job = New-AzVm -Name MyVMA3 -Credential $vmCredential -AsJob
-# $vm = Receive-Job $job -Wait
-# $vm
-# Write-Host "</async>"
+Describe 'New-AzVm' {
+    It 'WhatIf' {
+        New-AzVm -Name MyVM -Credential $vmCredential -WhatIf
+    }
+    It 'Create Windows VM' {
+        New-AzVm -Name MyVMA1 -Credential $vmCredential -ResourceGroupName Something1 -Verbose
```

Comment:
I think we should pair this with some validations, like validating the output of the cmdlet, and perhaps performing a Get and verifying proeprties for the expected resources.